src/components/Button/button.module.scss

```diff
@@ -18,14 +18,26 @@
 
   .icon + .button-text-large:not(:empty) {
     margin-left: $button-spacer-large;
+
+    + .icon-two {
+      margin-left: $button-spacer-large;
+    }
   }
 
   .icon + .button-text-medium:not(:empty) {
     margin-left: $button-spacer-medium;
+
+    + .icon-two {
+      margin-left: $button-spacer-medium;
+    }
   }
 
   .icon + .button-text-small:not(:empty) {
     margin-left: $button-spacer-small;
+
+    + .icon-two {
+      margin-left: $button-spacer-small;
+    }
   }
```

Comment:
need to account for RTL scenario also, in which case I presume the icons will switch sides?

Reply:
RTL is handled by the button styles itself. 
<img width="620" alt="image" src="https://user-images.githubusercontent.com/109717425/208874759-d323f064-b441-4209-bd94-5ad6368bb76e.png">